src/docker-compose-cli.js

```diff
@@ -31,7 +34,18 @@ const composeCommand = (filePaths, ...params) => {
   });
 };
 
-const pull = (fileName) => composeCommand(fileName, 'pull');
+const pull = async (fileName, retry = 100) => {
```

Comment:
Just an aside, but I really like starting with the max retries here and doing a burn-down simply because it lets any reader see at a glace how many times this function is going to retry (instead of starting at 0 and having to dig into the function code to see what is the max...